test/e2e-tests.sh

```diff
@@ -66,13 +68,16 @@ function set_embedded_status() {
 function run_e2e() {
   # Run the integration tests
   header "Running Go e2e tests"
-  go_test_e2e -timeout=20m ./test/... || failed=1
+  # Skip ./test/*.go tests if SKIP_GO_E2E_TESTS == true
+  if [ "${SKIP_GO_E2E_TESTS}" != "true" ]; then
+    go_test_e2e -timeout=${E2E_GO_TEST_TIMEOUT} ./test/... || failed=1
+  fi
 
   # Run these _after_ the integration tests b/c they don't quite work all the way
   # and they cause a lot of noise in the logs, making it harder to debug integration
   # test failures.
   if [ "${RUN_YAML_TESTS}" == "true" ]; then
-    go_test_e2e -parallel=4 -mod=readonly -tags=examples -timeout=20m ./test/ || failed=1
```

Comment:
Why is the parallel=4 flag removed? Does it mean it will run as many as possible in parallel?